### PR TITLE
android: fixed ijkj4a static libray linked.

### PR DESCRIPTION
**Pull request by chodison**

### CODE REVIEW

File: ios/IJKMediaPlayer/IJKMediaPlayer/ijkmedia/ijkplayer/ios/pipeline/h264_sps_parser.h [hunk 1]
```diff
@@ -264,7 +264,7 @@ static void parseh264_sps(uint8_t *sps, uint32_t sps_size,  int *level, int *pro
     *max_ref_frames = (int)sps_info.max_num_ref_frames;
 }
 
-bool validate_avcC_spc(uint8_t *extradata, uint32_t extrasize, int32_t *max_ref_frames, int *level, int *profile)
+static bool validate_avcC_spc(uint8_t *extradata, uint32_t extrasize, int32_t *max_ref_frames, int *level, int *profile)
```
Comment (xinzhengzhang) [inline ios/IJKMediaPlayer/IJKMediaPlayer/ijkmedia/ijkplayer/ios/pipeline/h264_sps_parser.h]: Has been fixed

File: ijkmedia/ijksdl/Android.mk [hunk 2]
```diff
@@ -70,8 +70,8 @@ LOCAL_SRC_FILES += android/ijksdl_vout_android_nativewindow.c
 LOCAL_SRC_FILES += android/ijksdl_vout_android_surface.c
 LOCAL_SRC_FILES += android/ijksdl_vout_overlay_android_mediacodec.c
 
-LOCAL_SHARED_LIBRARIES := ijkffmpeg ijkj4a
-LOCAL_STATIC_LIBRARIES := cpufeatures yuv_static
+LOCAL_SHARED_LIBRARIES := ijkffmpeg
+LOCAL_STATIC_LIBRARIES := cpufeatures yuv_static ijkj4a
```
Comment (xinzhengzhang) [inline ijkmedia/ijksdl/Android.mk]: why

Comment (chodison) [inline ijkmedia/ijksdl/Android.mk]: SEE: https://github.com/Bilibili/ijkplayer/blob/master/ijkmedia/ijkj4a/Android.mk
...
LOCAL_MODULE := ijkj4a
include $(BUILD_STATIC_LIBRARY)

$(call import-module,android/cpufeatures)

Comment (xinzhengzhang) [inline ijkmedia/ijksdl/Android.mk]: What's your point?

Comment (chodison) [inline ijkmedia/ijksdl/Android.mk]: include $(BUILD_STATIC_LIBRARY)

**Pull request merged**